docs/reference/data-streams/logs.asciidoc

```diff
@@ -46,154 +48,146 @@ DELETE _index_template/my-index-template
+
+If you need to customize the limit, you can override it at the mapping level or change the index level default. 
+
+[discrete]
+[[logs-db-ignore-limit]]
+==== `ignore_dynamic_beyond_limit`
 
 In `logsdb` index mode, the setting `index.mapping.total_fields.ignore_dynamic_beyond_limit` is set to `true` by
-default. This allows dynamically mapped fields to be added on top of statically defined fields without causing document
-rejection, even after the total number of fields exceeds the limit defined by `index.mapping.total_fields.limit`. The
-`index.mapping.total_fields.limit` setting specifies the maximum number of fields an index can have (static, dynamic
-and runtime). When the limit is reached, new dynamically mapped fields will be ignored instead of failing the document
-indexing, ensuring continued log ingestion without errors.
+default. This setting allows dynamically mapped fields to be added on top of statically defined fields, even when the total number of fields exceeds the `index.mapping.total_fields.limit`. Instead of triggering an index failure, additional dynamically mapped fields are ignored so that ingestion can continue.
 
-NOTE: When automatically injected, `host.name` and `@timestamp` contribute to the limit of mapped fields. When
-`host.name` is mapped with `subobjects: true` it consists of two fields. When `host.name` is mapped with
-`subobjects: false` it only consists of one field.
+NOTE: When automatically injected, `host.name` and `@timestamp` count toward the limit of mapped fields. If `host.name` is mapped with `subobjects: true`, it has two fields. When mapped with `subobjects: false`, `host.name` has only one field.
 
 [discrete]
 [[logsdb-nodocvalue-fields]]
 === Fields without doc values
 
-When `logsdb` index mode uses synthetic `_source`, and `doc_values` are disabled for a field in the mapping,
-Elasticsearch may set the `store` setting to `true` for that field as a last resort option to ensure that the field's
-data is still available for reconstructing the document’s source when retrieving it via
+When the `logsdb` index mode uses synthetic `_source` and `doc_values` are disabled for a field in the mapping,
+{es} might set the `store` setting to `true` for that field. This ensures that the field's
+data remains accessible for reconstructing the document's source when using
 <<synthetic-source,synthetic source>>.
 
-For example, this happens with text fields when `store` is `false` and there is no suitable multi-field available to
-reconstruct the original value in <<synthetic-source,synthetic source>>.
-
-This automatic adjustment allows synthetic source to work correctly, even when doc values are not enabled for certain
-fields.
+For example, this adjustment occurs with text fields when `store` is `false` and no suitable multi-field is available for
+reconstructing the original value.
 
 [discrete]
 [[logsdb-settings-summary]]
 === LogsDB settings summary
```

Comment:
```suggestion
=== Settings summary
```

Reply:
Subsequent to this, I made a [suggestion](https://github.com/elastic/elasticsearch/pull/118303#pullrequestreview-2492300187) about bigger rejiggering/renaming of sections